lib/msf/core/handler/reverse/comm.rb

```diff
@@ -39,9 +39,8 @@ def select_comm
     comm
   end
 
-  def via_string_for_ip(ip, comm)
+  def via_string(comm)
```

Comment:
@adfoster-r7 See https://github.com/rapid7/metasploit-framework/pull/15706#issuecomment-928656518, I think we should but I'm not sure if its relevant to this PR specifically, however since it is a standard naming convention, it would likely be good to update everything at once as you mentioned.